images/origin/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN ln -s /usr/bin/openshift /usr/bin/oc && \
     ln -s /usr/bin/openshift /usr/bin/openshift-deploy && \
     ln -s /usr/bin/openshift /usr/bin/openshift-docker-build && \
     ln -s /usr/bin/openshift /usr/bin/openshift-sti-build && \
-    ln -s /usr/bin/openshift /usr/bin/openshift-router && \
+    ln -s /usr/bin/openshift /usr/bin/openshift-f5-router && \
```

Comment:
Was this intentional - to change ```openshift-router``` to ```openshift-f5-router```?
I guess the main question really is why was this needed the origin dockerfile? Both the f5 and haproxy routers run inside a container and have the appropriate router binary in their images.